file-formats/doma/type/zif_aff_doma_v1.abap

```diff
@@ -0,0 +1,112 @@
+INTERFACE zif_aff_doma_v1 PUBLIC.
+
+  "! <p class="shorttext">Plus - Minus sign for domains</p>
+  "! Plus - Minus sign for domains
+  TYPES ty_sign                 TYPE c.
+
+  "! <p class="shorttext">Case sensiive flag for domains</p>
+  "! Case sensiive flag for domains
+  TYPES ty_case_sensitive       TYPE c.
+
+  "! <p class="shorttext">Data type for the domain</p>
+  "! Data type for the domain
+  TYPES ty_data_type            TYPE c LENGTH 4.
+
+  "! <p class="shorttext">No Characters</p>
+  "! Number of Characters
+  TYPES ty_numb_char            TYPE c LENGTH 6.
+
+  "! <p class="shorttext">Decimals</p>
+  "! Decimals
+  TYPES ty_decimals             TYPE c LENGTH 6.
+
+  "! <p class="shorttext">Output Length</p>
+  "! Output Length
+  TYPES ty_output_length        TYPE c LENGTH 6.
+
+  "! <p class="shorttext">Convertion Routine</p>
+  "! Convertion Routine
+  TYPES ty_conv_routine         TYPE c LENGTH 5.
+
+  "! <p class="shorttext">Value table</p>
+  "! Value table
+  TYPES ty_value_table          TYPE c LENGTH 30.
+
+  "! <p class="shorttext">Single Values</p>
+  "! Single Values
+  TYPES: 
+    BEGIN OF ty_single_value,
+      "! <p class="shorttext">Fixed value</p>
+      "! Value
+      fixed_value                 TYPE c LENGTH 10,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description                 TYPE zif_aff_types_v1=>ty_header_60_src,
```

Comment:
This seems to be the wrong type. I guess you want to use `ty_description_60` don't you.
```suggestion
      description                 TYPE zif_aff_types_v1=>ty_description_60,
```

Reply:
Yes, you are correct.